src/main/features/dashboard/views/macro/caseHistory.njk

```diff
@@ -0,0 +1,76 @@
+{% from "./caseStatus.njk" import caseStatusForClaimant, caseStatusForDefendant %}
+{% from "timeRemaining.njk" import timeRemaining %}
+
+{% macro caseHistoryForClaimant(claim, url) %}
+  {% for state in claim.stateHistory %}
+    <div class="status-block">
+      {% if state.status === ClaimStatus.CLAIM_REJECTED %}
+        <h3 class="heading-small">{{ t('The defendant’s response') }}</h3>
+          <p>
+            {{ t('{{ defendantName }} has rejected your claim. We’ll send you a directions questionnaire – this is a form which you must complete to tell us more about the claim.',
+              { defendantName: claim.claimData.defendant.name }) }}
+          </p>
+          {# TODO: Check whether response PDF can be downloaded by claimant #}
```

Comment:
Clarified with Marc and Art that claimant should have an option to download full response PDF. Will make a change before that PR is merged.

Reply:
Done.